app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<h>Hello World</h>
+<main class="home">
+  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+  <%= render 'svg/home-ill-cup' %>
+  <section class="home__section">
+      <h1>Created for tired people, by slightly less tired people.</h1>
```

Comment:
Hmm...  In this one case, I do think the `Ground Rules` text is the appropriate `<h1>` or there should maybe be a hidden `<h1>` that is "Ground Rules Homepage" or something.  That's where you are, you're on the Homepage, and the page title (h1) should make that clear.